plugin/pkg/scheduler/api/types.go

```diff
@@ -43,6 +43,7 @@ type PriorityPolicy struct {
 	// For the Kubernetes provided priority functions, the name is the identifier of the pre-defined priority function
 	Name string `json:"name"`
 	// The numeric multiplier for the minion scores that the priority function generates
+	// The weight should be non-zero and can be a positive or a negative integer
```

Comment:
Is there a human-friendly description of the types in this file? The names alone aren't quite clear enough for me to follow.